geometry/clip-geometry/src/main/java/com/esri/samples/clip_geometry/ClipGeometrySample.java

```diff
@@ -116,18 +116,28 @@ public void start(Stage stage) {
       // create a button to perform the clip operation
       Button clipButton = new Button("Clip");
```

Comment:
since this button is now handling clip and reset, a better variable for this can now just be button

```suggestion
      var button = new Button("Clip");
```